clang/utils/TableGen/ClangDiagnosticsEmitter.cpp

```diff
@@ -886,6 +909,11 @@ struct DiagTextDocPrinter : DiagTextVisitor<DiagTextDocPrinter> {
       makeRowSeparator(RST[I]);
   }
 
+  void VisitEnumSelect(EnumSelectPiece *P) {
+    // For now, document this as if it were a 'select'.
```

Comment:
Should we file an issue to address this?

Reply:
I don't think so?  Actually, the 'For now,' could probably go away.  The documentation page seems more directed at someone 'reading' the documentation, so the "list of possible things this could say" is likely sufficient.  WDYT?

Reply:
I think that makes sense -- let's drop the "for now" in the comment and just explain that in terms of documentation, there's no difference between select and enum_select.